src/components/routes/CalendarItemDisplay.vue

```diff
@@ -0,0 +1,241 @@
+            />
+          </svg>
+        </q-icon>
+      </template>
+      <!-- Background: From work -->
+      <template v-if="direction === 'fromWork'">
+        <!-- Icon: From work - active -->
+        <q-icon
+          v-if="active"
+          color="primary"
+          class="full-width full-height absolute-full"
+          data-cy="calendar-item-icon-fromwork-active"
+        >
+          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 135 38">
```

Comment:
Should be standalone SVG image.

Reply:
Refactored using [public svg with svg:use](https://quasar.dev/vue-components/icon#svg-use-way)
Will be used for new custom SVGs.